forem/forem
# Add Staging specific seed file

## What type of PR is this? (check all applicable)

## Description
Created a speed file that is specific to the preview environment.

## Related Tickets & Documents
Mostly related to https://github.com/forem/forem/issues/19264
Touches a bit of https://github.com/forem/forem-internal-eng/issues/602

## QA Instructions, Screenshots, Recordings
Please visit the Uffizzi preview, you should notice way more articles and user count because I've upped `SEEDS_MULTIPLIER`

You can also visit the seed logs itself here https://app.uffizzi.com/projects/4723/deployments/24701/containers/118758/logs/current
## Added tests?

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
n/a
